xla/service/gpu/stream_attribute_annotator.h

```diff
@@ -45,6 +45,8 @@ namespace xla::gpu {
 
 class StreamAttributeAnnotator : public HloModulePass {
  public:
+  explicit StreamAttributeAnnotator(bool copy_start_done = false)
```

Comment:
Nit: add a description of what the var is doing.

Reply:
> Nit: add a description of what the var is doing.

Done.